notebooks/plot_backend.py

```diff
@@ -78,8 +78,8 @@
 
 # %%
 # Define directory to VIO output csv files as well as ground truth absolute poses.
-vio_output_dir = "/home/tonirv/datasets/kimera_rig/output_logs/"
-gt_data_file = ""
+vio_output_dir = ""
+gt_data_file = vio_output_dir + "traj_gt.csv"
```

Comment:
I'd consider using `pathlib` or `os.path.join` here (makes it less error-prone to remember if you need a trailing slash or not). My preference would be `pathlib`, which is in the standard library in python 3 (which we should consider mandating the use of)

Reply:
@marcusabate : let me know if you want to do this, otherwise I'll go ahead and merge